docs/guide/workspace.md

```diff
@@ -172,6 +172,30 @@ export default mergeConfig(
 ```
 :::
 
+At the `defineWorkspace` level you can use the `extends` option to inherit from your root-level config.
```

Comment:
Just noticed, but how is this different from 

https://github.com/CoolGoose/vitest/blob/266330c3dee2ff55ec1fc1aad1fdc03aff2a9b6b/docs/guide/workspace.md?plain=1#L62

Reply:
It's not fully different, I wrote it mostly in the context of 'please make sure you understand there's no base config automatically extended when you're doing workspaces' a bit localized to the section that talks about Configuration specifically.

Looking back on how I wrote it it's a bit clunky indeed.

The problem is that conceptually, I think it was more of a case of expecting the workspaces to automatically extend the vitest.config.ts (especially in the context that not all options are available in the workspace config, so you'll need to use extends). and also the difference of `mergeConfig` for vitest and `.extends` in the workspace 😅 

Happy to just close this if you think it's just a bit more confusing.

Reply:
I see, I guess it makes sense to also duplicate an example with `extends` here alongside `mergeConfig` as an alternative